### PR TITLE
Add Darwin

### PR DESCRIPTION
**Pull request by nithinkodadhavadee**

### CODE REVIEW

**Comment by Richienb** [comment]
@nithinkodadhavadee Can you send a link to a page featuring the Darwin programming language?

**Comment by nithinkodadhavadee** [comment]
https://people.inf.ethz.ch/gonnet/DarwinManual/node4.html

the documentation for the language.

**Comment by Richienb** [comment]
Should it be [`printf`](https://people.inf.ethz.ch/gonnet/DarwinManual/node35.html) instead of `lprint`?

**Comment by nithinkodadhavadee** [comment]
Well, there are ```[print](https://people.inf.ethz.ch/gonnet/DarwinManual/node22.html)``` and a ```[lprint](https://people.inf.ethz.ch/gonnet/DarwinManual/node22.html)``` statement too. SoI guess everything works.

**Comment by nithinkodadhavadee** [comment]
hey @Richienb, is this pull request valid or should I close it? Or are there any changes to do here? I have this thing where I don't like it when a pull request is left open and nothing is going on.

**Comment by nithinkodadhavadee** [comment]
and yes I have a question. In this pull request, I can see some older commits that I made which are already merged to the repository. Is this something that happens usually(a thing that I haven't noticed anywhere) so did I do something wrong?
image

**Comment by Richienb** [comment]
@nithinkodadhavadee 

> Is this something that happens usually(a thing that I haven't noticed anywhere) so did I do something wrong?

**Review by Richienb — CHANGES_REQUESTED** [review]
Shouldn't the file have a `.drw` extension? See: https://en.wikipedia.org/wiki/Darwin_(programming_language)

**Comment by nithinkodadhavadee** [comment]
> Shouldn't the file have a `.drw` extension? See: https://en.wikipedia.org/wiki/Darwin_(programming_language)

sorry I didn't notice the extension. I also changed the print function to printf(), most people refer the Wikipedia than the lecture notes, so changed it to what people might notice first.

**Comment by nithinkodadhavadee** [comment]
> @nithinkodadhavadee
> 
> > Is this something that happens usually(a thing that I haven't noticed anywhere) so did I do something wrong?

I'm sorry I don't understand, you haven't typed anything.

**Comment by Richienb** [comment]
@nithinkodadhavadee There was supposed to be a comment but somehow it's not there. I've updated it now.

File: d/Darwin.drw [hunk 1]
```diff
@@ -0,0 +1 @@
+printf('Hello World")
```
Comment (Richienb) [inline d/Darwin.drw:1]: Which type of quote are we using, `'` or `"`?

**Review by Richienb — CHANGES_REQUESTED** [review]

**Pull request merged**